### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by felddy**
# Lineage Pull Request: CONFLICT #

DANGER

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Clone the `lineage/skeleton` branch locally:

    ```console
    git clone --single-branch --branch lineage/skeleton git@github.com:cisagov/nessus-packer.git
    ```

1. Review each of the following files looking for [conflict-resolution markers](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts):

    ```console
    .github/workflows/prerelease.yml
    .github/workflows/release.yml
    .gitignore
    README.md
    src/packer.json
    terraform-build-user/versions.tf
    terraform-post-packer/main.tf
    terraform-test-user/versions.tf
    terraform/versions.tf
    ```

1. Resolve each of the conflicts and `add` your changes to the branch:

    ```console
    git add .github/workflows/prerelease.yml .github/workflows/release.yml .gitignore README.md src/packer.json terraform-build-user/versions.tf terraform-post-packer/main.tf terraform-test-user/versions.tf terraform/versions.tf 
    ```

1. Once all the changes have been added, `commit` and `push` the changes back to
GitHub:

    ```console
    git commit --message="Resolved Lineage conflicts."
    git push
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: terraform-build-user/backend.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-build-user.tfstate"
```
Comment (jsf9k) [inline terraform-build-user/backend.tf]: This key needs to be updated.

File: terraform-build-user/main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  providers = {
+    aws                       = aws
+    aws.images-production-ami = aws.images-production-ami
+    aws.images-staging-ami    = aws.images-staging-ami
+    aws.images-production-ssm = aws.images-production-ssm
+    aws.images-staging-ssm    = aws.images-staging-ssm
+  }
+
+  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*"]
+  user_name      = "build-skeleton-packer"
```
Comment (jsf9k) [inline terraform-build-user/main.tf]: The user name and application need to be updated, and possibly the list of SSM ParameterStore parameters.

File: terraform-post-packer/backend.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-post-packer.tfstate"
```
Comment (jsf9k) [inline terraform-post-packer/backend.tf]: This key needs to be updated.

**Review by jsf9k — CHANGES_REQUESTED** [review]
The terraform directories seem incorrect here.

File: terraform-build-user/backend.tf (hunk 1 above)
Comment (dav3r) [reply terraform-build-user/backend.tf]: Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/29fe68a31f29ec3fc4578fbba4c10567901424a8.

File: terraform-build-user/main.tf (hunk 2 above)
Comment (dav3r) [reply terraform-build-user/main.tf]: Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/aafcf86446a602221f477746b89be07d462dfdc3.  I verified that the list of SSM parameters did not need to be changed.

File: terraform-post-packer/backend.tf (hunk 3 above)
Comment (dav3r) [reply terraform-post-packer/backend.tf]: Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/81625b9ef2298d0f606ae87cb100f5a4109cfcaf.

**Comment by dav3r** [comment]
This PR should be ready for review now.

After I made the latest round of changes, here's what I did to get this repo up to snuff:
- Successfully ran `terraform destroy` on the old `terraform-test-user` resources
- Successfully ran `terraform apply` for the new `terraform-build-user`
- Successfully ran `terraform apply` for the updated `terraform-post-packer`
- Successfully ran `terraform-to-secrets` to update all of the secrets in this repo

**Comment by felddy** [comment]
For what it's worth @dav3r ... you can add your approval to this PR.

**Review by dav3r — APPROVED** [review]
👍 👍 👍

**Review by jsf9k — APPROVED** [review]

**Pull request merged**